src/pybind/mgr/cephadm/module.py

```diff
@@ -2336,6 +2339,43 @@ def create_self_signed_cert() -> Tuple[str, str]:
         })
         return config_file
 
+    def _get_dashboard_url(self):
+        # type: () -> str
```

Comment:
Wouldn't that return either `str` or `None`?

Reply:
ah yes, true